pkgs/tools/security/bitwarden-directory-connector/default.nix

```diff
@@ -44,7 +44,8 @@ let
     ];
 
     nativeBuildInputs = [
-      python3
+      # gyp from node_modules wants to import distutils
+      python311
```

Comment:
```suggestion
      (python3.withPackages (ps: with ps; [ setuptools ]))
```

Reply:
Why do we need that? Seems to work without it.

Reply:
Because then it'll probably work with Python 3.12.

Reply:
apperantly it does. I though distutils is gone for good.

Reply:
It is, but the script falls back to distutils if it can't find setuptools.